.travis.yml

```diff
@@ -1,7 +1,8 @@
 language: node_js
 node_js:
-    - "6"
-    - "node"
+- 7 # Current stable
```

Comment:
Need is to avoid that someone who opens a PR right after Node v8 gets released has build automatically run against v8 silently and potentially breaks for no obvious reason. Explicit is better in this case, as it keeps things clear and predictable. It definitely outweighs maintenance cost 😅